SimulationAgent/DeviceConnection/DeviceConnectionActor.cs

```diff
@@ -314,25 +345,28 @@ public void HandleEvent(ActorEvents e)
             switch (e)
             {
                 case ActorEvents.Started:
-                    if (this.loopSettings.SchedulableFetches <= 0) return;
-                    this.loopSettings.SchedulableFetches--;
+                    //if (this.loopSettings.SchedulableFetches <= 0)
+                    //    return;
+                    //this.loopSettings.SchedulableFetches--;
```

Comment:
are these comments here for any reason? if not, please remove

Reply:
Fixed.